src/datreant/core/rsync.py

```diff
@@ -33,11 +34,16 @@ def rsync(source, dest, compress=True, backup=False, dry=False, checksum=True,
         every other path is excluded
     exclude: str or list
         Paths to be excluded from the copy
+    overwrite: bool
+        If True, replace the files in `dest` regardless of timestamp
```

Comment:
Could you add a bit more, something like "If Fase, files in `dest` that are newer than files in `source` will not be overwritten"

I wasn't sure exactly what effect this had until I looked at your test.